### PR TITLE
ONNX E2E add filtering launchers

### PR DESCRIPTION
**Pull request by kshpv**
### Changes

Add launcher filtering in run_ptq by adding 'target_frameworks' option
Removed non used script perfomance_checker

### Reason for changes

96627

### Related tickets

96627

### Tests

No

### CODE REVIEW

**Review by sstrehlk — APPROVED** [review]

**Review by KodiaqQ — APPROVED** [review]

File: tests/onnx/benchmarking/run_ptq.py [hunk 1]
```diff
@@ -88,12 +94,32 @@ def run(onnx_model_path: str, output_model_path: str, dataset: nncf.Dataset,
...
+    """
+    if len(config[mode]) < 2:
+        return
+    to_remove_indices = []
+    for i, config_entry in enumerate(config[mode]):
+        if config_entry.get(AccuracyCheckerConstansts.LAUNCHERS)[0].get(
+                AccuracyCheckerConstansts.FRAMEWORK) != AccuracyCheckerConstansts.ONNXRUNTIME_KEY:
+            to_remove_indices.append(i)
+    for index in sorted(to_remove_indices, reverse=True):
+        del config[mode][index]
```
[the remark this answers is not in the document]
Comment (kshpv) [reply tests/onnx/benchmarking/run_ptq.py]: Removed this logic. Now filtering is done p adding an option for AC

[the remark this answers is not in the document]
Comment (vshampor) [reply tests/onnx/benchmarking/run_ptq.py]: Adjust PR title and description accordingly.

[the remark this answers is not in the document]
Comment (kshpv) [reply tests/onnx/benchmarking/run_ptq.py]: Done

**Review by vshampor — APPROVED** [review]

**Comment by AlexKoff88** [comment]
I see `subprocess.Popen` is used. @vshampor, why did you approve this?

**Comment by vshampor** [comment]
> I see `subprocess.Popen` is used. @vshampor, why did you approve this?

I guess I'll have to request a direct quote here because I can't see this line in the diff no matter how hard I look.

**Pull request merged**